src/vim9class.c

```diff
@@ -269,14 +269,10 @@ object_index_from_itf_index(class_T *itf, int is_method, int idx, class_T *cl,
     {
 	// TODO: Need a table for fast lookup?
 	char_u *name = itf->class_class_members[idx].ocm_name;
-	for (int i = 0; i < i2c->i2c_class->class_class_member_count; ++i)
-	{
-	    ocmember_T *m = &i2c->i2c_class->class_class_members[i];
-	    if (STRCMP(name, m->ocm_name) == 0)
-	    {
-		return i;
-	    }
-	}
+	int	m_idx = class_member_idx(i2c->i2c_class, name, 0);
+	if (m_idx >= 0)
+	    return m_idx;
```

Comment:
In object_index_from_itf_index, the `is_static` param can be removed I think. It was added to support accessing a static with an object. So this branch can go away.

Reply:
Took a quick look. To actually take `is_static` out, would have to start where it's first used and go forward. That can be done as a cleanup, in a post release time frame (assuming accessing static with an object doesn't come back).